src/EventListener/EndEmptySessionSubscriber.php

```diff
@@ -3,14 +3,15 @@
 namespace eLife\Journal\EventListener;
 
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
 final class EndEmptySessionSubscriber implements EventSubscriberInterface
 {
     public static function getSubscribedEvents() : array
     {
-        return [KernelEvents::RESPONSE => ['onKernelResponse', -1000]];
+        return [KernelEvents::RESPONSE => ['onKernelResponse', -127]]; # Before TestSessionListener
```

Comment:
Only affect the tests, as `$session->save()` is called otherwise, which marks the session as _not_ started.